config/authorities/linked_data/loc.json

```diff
@@ -1,30 +1,28 @@
 {
   "QA_CONFIG_VERSION": "2.1",
   "prefixes": {
-    "loc": "http://id.loc.gov/vocabulary/identifiers/",
-    "skos": "http://www.w3.org/2004/02/skos/core#",
-    "madsrdf": "http://www.loc.gov/mads/rdf/v1#",
-    "owl": "http://www.w3.org/2002/07/owl#"
+    "loc":     "http://id.loc.gov/vocabulary/identifiers/",
+    "madsrdf": "http://www.loc.gov/mads/rdf/v1#"
```

Comment:
Most of the changes in this file are white space changes for readability.  There are two actual changes.  One here to remove prefixes that are already defined in the ldpath gem.  It is redundant to define them here.